spec/support/json_schemas.rb

```diff
@@ -4,8 +4,11 @@
   def load_schema(file_name, version: 'v1')
     return unless File.file?("#{Rails.root}/swagger/#{version}/#{file_name}")
 
-    schema = load_yaml_schema(file_name)
-    JSON::Validator.add_schema(JSON::Schema.new(schema, file_name))
+    schema = load_yaml_schema(file_name, version: version)
+
+    # TODO: when moving to fully support v1 and v2 stop loading v1 twice
```

Comment:
When adding documentation for `V2`, we have to reference some "shared" documentation in `V1`, example: `'../v1/pagination_parameters.yaml#/PerPage'`. This works well when we swaggerize our schemas, and is compliant with swagger. However when we add schemas to `JSON::Validator`, we need to add them under the correct path or it won't be able to parse them. For instance it would attempt to read `'../v1/pagination_parameters.yaml#/PerPage'` and try to go up a directory to `v1` which is not supported before. If we were to add paths for v1 and v2, this would mean we would have to reference things everywhere with `v1` to support it. For now until we fully support v2, load v1 under the root path and v1.